src/Device/AbstractDevice.php

```diff
@@ -190,7 +184,7 @@ protected function sanitizeInput($input)
      *
      * @return array
      */
-    protected function createProcessArguments(Input $input)
+    protected function createProcessArguments(Input $input, string $version)
```

Comment:
Please do not add unused `$version` argument. You can `$this->ghostscript->getVersion()` when necessary.
```suggestion
    protected function createProcessArguments(Input $input)
```

Reply:
If you do not mind doble calling the getVersion then Ok, ill remove the BC :)

Reply:
This is no problem, because the version will be cached in as static string.

Reply:
No worries, I get it :)